ksa_compliance/patches/_2024_08_19_update_old_fatoora_url_in_business_settings.py

```diff
@@ -0,0 +1,32 @@
+import frappe
+
+
+def execute():
+    print("Start updating old fatoora url")
+    sql = """
+            SELECT company, fatoora_server_url
+            FROM `tabZATCA Business Settings`
+        """
+    old_urls = frappe.db.sql(sql, as_dict=True)
+    fatoora_servers = {
+        "sandbox": "Sandbox",
+        "simulation": "Simulation",
+        "production": "Production"
+    }
+    print("Updating Fatoora Server based on old urls")
+    for url in old_urls:
+        if url["fatoora_server_url"].startswith('https://gw-fatoora.zatca.gov.sa/e-invoicing/developer-portal'):
+            fatoora_server = fatoora_servers["sandbox"]
+        elif url["fatoora_server_url"].startswith('https://gw-fatoora.zatca.gov.sa/e-invoicing/simulation'):
+            fatoora_server = fatoora_servers["simulation"]
+        elif url["fatoora_server_url"].startswith('https://gw-fatoora.zatca.gov.sa/e-invoicing/core'):
+            fatoora_server = fatoora_servers["production"]
+        else:
+            fatoora_server = None
+        if fatoora_server:
```

Comment:
Let's add a print statement here stating that we're setting the server for a given ID to the computed value